emailproxy.py

```diff
@@ -67,7 +68,7 @@
 no_gui_parser.add_argument('--no-gui', action='store_true')
 no_gui_parser.add_argument('--external-auth', action='store_true')
 no_gui_args = no_gui_parser.parse_known_args()[0]
-if not no_gui_args.no_gui:
+if not no_gui_args.no_gui and 'setuptools.config.pyprojecttoml' not in sys.modules:  # don't import when building wheel
```

Comment:
I opened https://github.com/simonrob/email-oauth2-proxy/pull/201 to address this. I hadn't caught this since I had to install all the requirements when playing around with things (I didn't want to deal with the import errors), but since this came up I figured I should address it before continuing.